workflow.py

```diff
@@ -0,0 +1,27 @@
+import utils
+from search.search_engine import SearchEngine
+import constants
+
+def query_workflow(search, query):
+    short_gresponse = utils.from_gcloud_remove_extra(utils.gcloud_syntax_extraction(query))
+
+    keywords = []
+
+    for token in short_gresponse['tokens']:
+        keywords.append(token['lemma'])
+
+    lan = short_gresponse['lan']
+    synonyms = []
+    if lan == 'es':
```

Comment:
Are you guys planning on supporting languages other than spanish for now? If not, you can probably move this language check to near the top of this function as a base case, and return early if it's not spanish.